packages/api/cms-api/src/common/validators/is-slug.ts

```diff
@@ -16,7 +16,7 @@ export const IsSlug = (validationOptions?: ValidationOptions) => {
 export class IsSlugConstraint implements ValidatorConstraintInterface {
     async validate(value: string): Promise<boolean> {
         // Regex matches unreserved characters and percent encoding (see https://tools.ietf.org/html/rfc3986#section-2.1)
-        return /^([a-zA-Z0-9-._~]|%[0-9a-fA-F]{2})+$/.test(value);
+        return /^([a-zA-Z0-9-._~]|%[0-9a-fA-F]{2})+$/.test(decodeURIComponent(value));
```

Comment:
```suggestion
        return /^([a-zA-Z0-9-._~])+$/.test(value);
```

Wouldn't it be better to remove support for URL encoded characters from the regex instead of transforming the value? I don't think it's a good practice to transform a value _only_ for validation (as the untransformed would be stored if the validation was faulty)

Reply:
i have changed the regex to not decode the slug anymore and instead explicitly blocks the percent encoding for "/" %2F